airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/conftest.py

```diff
@@ -186,6 +207,25 @@ def docker_runner_fixture(
     )
 
 
+@pytest.fixture(autouse=True)
```

Comment:
```suggestion
@pytest.fixture()
```

If I'm not mistaken `autouse` means the fixture code will always get executed, even if no test depend on it. I'm not sure this is what we want.

Reply:
Going to try `scope="function"` instead, which may be more like what we want.

Reply:
@alafanechere rather than passing in the `setup_and_teardown` fixture to each test where it might be needed, WDYT about having `autouse=True` here? The advantage is that it centralizes the logic for whether to run setup/teardown to the `acceptance-test-config.yml`. For any tests for which setup/teardown config isn't present, the fixture will be a noop, and we won't have to remember to mark or add the fixture to new tests.

Since DB sources is already going to have to add the setup config to the tests for which it's required in `acceptance-test-config.yml`, this feels like the cleanest approach.

Reply:
🆗 . In any case we don't have to make it perfect right away and I don't have a better suggestion, let's ship it on figure out how well it's working for our use cases.